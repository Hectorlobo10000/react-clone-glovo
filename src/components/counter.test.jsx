import React from "react";
import { render } from "@testing-library/react";

import Counter from "./counter";

test("Counter component", () => {
  const { asFragment } = render(<Counter />);
  <Counter
    handleDecrease={jest.fn()}
    handleIncrease={jest.fn()}
    quantity="0"
  />;

  expect(asFragment()).toMatchSnapshot();
});
