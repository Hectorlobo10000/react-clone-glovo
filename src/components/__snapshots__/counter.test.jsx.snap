// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter component 1`] = `
<DocumentFragment>
  .emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 10px;
}

.emotion-0 {
  border: 2px solid black;
  background-color: red;
  padding: 5px 10px;
  color: white;
}

.emotion-1 {
  padding: 5px 10px;
  font-size: 1.3em;
  font-family: Arial;
}

.emotion-2 {
  border: 2px solid black;
  background-color: green;
  padding: 5px 10px;
  color: white;
}

<div
    class="emotion-3"
  >
    <button
      class="emotion-0"
    >
      -
    </button>
    <span
      class="emotion-1"
    />
    <button
      class="emotion-2"
    >
      +
    </button>
  </div>
</DocumentFragment>
`;
